### PR TITLE
Fix: desert flora can now be cut down

### PR DESCRIPTION
**Pull request by JohnDoe50974**
## About The Pull Request
Desert tallgrass and bushes (the orange-map thematic ones, found on Kutjevo and Trijent) could not have been cut with machetes/knives beforehand. This fixes that.

## Why It's Good For The Game
Bugfix good!

## Changelog

:cl: 50RemAndCounting
fix: Kutjevo/Trijent tallgrass and other flora can now be properly cut away with machetes and other bladed equipment.
/:cl:

### CODE REVIEW

**Review by Agameofscones — CHANGES_REQUESTED** [review]
discard changes to maps.txt, otherwise lgtm

**Comment by JohnDoe50974** [comment]
> discard changes to maps.txt, otherwise lgtm

Fixed

**Comment by Superwelder** [comment]
This makes me happy. Even when Kutjevo was added, we could chop the grass on LV so the discrepancy has always existed. Props for making it happen.

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -220,6 +220,7 @@ ICE GRASS
 /obj/structure/flora/grass/tallgrass/desert/corner
 	icon_state = "tallgrass_corner"
 	overlay_type = "tallgrass_overlay_corner"
+	cut_level = PLANT_CUT_MACHETE
```
Comment (stanalbatross) [inline code/game/objects/structures/flora.dm:223]: this needs to be on the parent obj not the /corner subtype

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**